features/step_definitions/coronavirus_local_restrictions_steps.rb

```diff
@@ -0,0 +1,7 @@
+When "I visit the postcode checker" do
+  visit_path "/find-coronavirus-local-restrictions"
+end
+
+Then "I should be redirected to the national restrictions page" do
+  expect(current_path).to eql("/guidance/new-national-restrictions-from-5-november")
```

Comment:
I’m surprised this passed in integration, as the checker is being redirected to the production page regardless of environment 🤔

Reply:
Unless `current_path` is doing some magic?

Reply:
Ah! `current_path` is stripping all domain information: https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession:current_path